released/usecases/services_apps/sap_build_code/step2/README.md

```diff
@@ -0,0 +1,23 @@
+# Setting up a sub account with the SAP AI Core service deployed
```

Comment:
This README should be updated to refelct step 2. maybe it even makes sense to have one genrich README describing step 1 and step 2 in the root folder

Reply:
Makes sense.